int16 record_flag
float32 theta_m1
float32 theta_l1
float32 delta_theta_r1
float32 Trr_ad
float32 m1_target
float32 theta_m2
float32 theta_l2
float32 delta_theta_r2
float32 Trr_cf
float32 m2_target
